http://sourceware.org/ml/gdb-patches/2012-06/msg00050.html
Subject: [patch] dejagnu compat. - missing find_go_linker  [Re: [patch, doc RFA] Go language support]

On Wed, 25 Apr 2012 04:17:35 +0200, Doug Evans wrote:
> +if {[info procs find_go_linker] == ""} {
> +    rename gdb_find_go find_go
> +    rename gdb_find_go_linker find_go_linker
> +    # No need to set use_gdb_compile.
> +}

Is there a reason for it?  With recent Fedora 17 update
	https://bugzilla.redhat.com/show_bug.cgi?id=635651

dejagnu has started to support 'find_gfortran'.  But it still does not support
'find_go_linker'.  This has resulted in regression failing to compile any
gdb.go/*.exp files.


Thanks,
Jan


gdb/testsuite/
2012-06-02  Jan Kratochvil  <jan.kratochvil@redhat.com>

	* lib/future.exp: Set $use_gdb_compile even if only find_go_linker is
	missing.

diff --git a/gdb/testsuite/lib/future.exp b/gdb/testsuite/lib/future.exp
index 40456c0..bf47988 100644
--- a/gdb/testsuite/lib/future.exp
+++ b/gdb/testsuite/lib/future.exp
@@ -514,7 +514,7 @@ if {[info procs find_gfortran] == ""} {
 if {[info procs find_go_linker] == ""} {
     rename gdb_find_go find_go
     rename gdb_find_go_linker find_go_linker
-    # No need to set use_gdb_compile.
+    set use_gdb_compile 1
 }
 
 if {$use_gdb_compile} {

